src/libcommon/comm.h

```diff
@@ -158,7 +161,8 @@ typedef enum {
     SIGNAL_NUM_UTILITY_ERROR_ADDED,
     SIGNAL_NUM_UTILITY_ERRORS_CLEARED,
     SIGNAL_NUM_UTILITY_SHOW_SETTINGS,
-    SIGNAL_NUM_UTILITY_SHOW_SYNTHESIS
+    SIGNAL_NUM_UTILITY_SHOW_SYNTHESIS,
+    SIGNAL_NUM_UTILITY_LOG_UPLOAD_STATUS_UPDATED,
```

Comment:
Isn't it pertinent to group it with the other new entries since they are about the same thing?